app/src/main/java/com/mapbox/search/demo/api/ForwardGeocodingBatchResolvingJavaExampleActivity.java

```diff
@@ -66,11 +55,7 @@ public void onError(@NonNull Exception e) {
     private final SearchMultipleSelectionCallback multipleSelection = new SearchMultipleSelectionCallback() {
 
         @Override
-        public void onResult(
-            @NonNull List<? extends SearchSuggestion> suggestions,
-            @NonNull List<? extends SearchResult> results,
-            @NonNull ResponseInfo responseInfo
-        ) {
+        public void onResult(@NonNull List<? extends SearchSuggestion> suggestions, @NonNull List<? extends SearchResult> results, @NonNull ResponseInfo responseInfo) {
```

Comment:
nit
Does this line fit to 120 characters line length?

Reply:
Discussed internally. Agreed to fix this issue in next release.